src/cls/rbd/cls_rbd_client.h

```diff
@@ -478,6 +478,10 @@ void mirror_image_status_get_summary_start(
 int mirror_image_status_get_summary_finish(
     ceph::buffer::list::const_iterator *iter,
     std::map<cls::rbd::MirrorImageStatusState, int32_t> *states);
+int mirror_image_status_remove(librados::IoCtx *ioctx,
+                               const std::string &global_image_id);
+void mirror_image_status_remove(librados::ObjectWriteOperation *op,
+                                const std::string &global_image_id);
```

Comment:
nit: Please also remove the comment "NOTE: deprecated - remove this method after Octopus is unsupported" in cls_rbd.cc.